core/org.openjdk.jmc.common/pom.xml

```diff
@@ -1,27 +1,27 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!--   
+<!--
    Copyright (c) 2018, Oracle and/or its affiliates. All rights reserved.
```

Comment:
year

Reply:
It's just whitespace. I thought I wouldn't update copyright in that case...